sormas-backend/src/main/java/de/symeda/sormas/backend/caze/surveillancereport/SurveillanceReportService.java

```diff
@@ -44,4 +49,25 @@ public Predicate buildCriteriaFilter(SurveillanceReportCriteria criteria, Criter
 
 		return filter;
 	}
+
+	public List<SurveillanceReport> getByCaseUuid(String caseUuid) {
+		CriteriaBuilder cb = em.getCriteriaBuilder();
+		CriteriaQuery<SurveillanceReport> cq = cb.createQuery(SurveillanceReport.class);
+		Root<SurveillanceReport> sampleRoot = cq.from(SurveillanceReport.class);
+		Join<SurveillanceReport, Case> caseJoin = sampleRoot.join(SurveillanceReport.CAZE, JoinType.LEFT);
+
+		cq.where(caseJoin.get(AbstractDomainObject.UUID).in(caseUuid));
+		return em.createQuery(cq).getResultList();
+	}
+
+	public List<SurveillanceReport> getByCaseUuids(List<String> caseUuids) {
+
+		CriteriaBuilder cb = em.getCriteriaBuilder();
```

Comment:
Add an if statement that returns an empty list if caseUuids is null or empty to avoid exceptions